### DISCUSSION TITLE
Actividad 3 Grupo 13

### DISCUSSION THREAD

**Discussion by pilita-acosta** [frlp-utn-ingsoft/catedra#133] [Show and tell]
https://github.com/facularrauri/vetsoft

**Comment by RodrigoJacznik** [comment]
Hola chicos. Antes que nada perdón por la demora.

Revise la app hasta el commit ab47870b734defae68c4602d75accc4a7c8a060f.

El trabajo esta muy bien, tanto los commits como los nombres de los branches (exceptuando `medicine` que no sigue el formato).
Saludos.
